website_event_questions_template/models/event_question_template.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class EventQuestionTemplateQuestion(models.Model):
+    _name = 'event.question.template.question'
+
+    title = fields.Char(required=True, translate=True)
+    sequence = fields.Integer(default=10)
+    is_individual = fields.Boolean(
+        string='Ask each attendee',
+        help="If True, this question will be asked for every attendee of a "
+             "reservation. If not it will be asked only once and its value "
+             "propagated to every attendees.",
+    )
+    answer_ids = fields.One2many(
+        comodel_name='event.question.template.answer',
+        inverse_name='question_id',
+        string="Answers",
+        required=True,
+    )
+    template_id = fields.Many2one(
+        comodel_name='event.question.template',
+        string='Event Question Template',
+        required=True,
+        ondelete='cascade',
+    )
+
+
+class EventQuestionTemplateAnswer(models.Model):
+    _name = 'event.question.template.answer'
+    _order = 'sequence,id'
+
+    name = fields.Char(
```

Comment:
Inherit the same way from `event.answer`. With this, new fields added via extra modules are also added here.